ci/concourse/azure-template-updater.yml

```diff
@@ -40,10 +46,54 @@ jobs:
     public: true
     plan:
       - get: git-repo-fortigate-autoscale-azure
+      - get: git-repo-fortigate-autoscale-azure-updated
       - get: resource-azure-list-fortigate-images-byol
       - get: resource-azure-list-fortigate-images-payg
       - get: resource-azure-list-fortianalyzer-images-byol
-      - task: update-azure-template
+      - task: git-checkout-head-branch
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: alpine/git
+          inputs:
+            - name: git-repo-fortigate-autoscale-azure
+            - name: git-repo-fortigate-autoscale-azure-updated
+          outputs:
+            - name: git-repo-fortigate-autoscale-azure-updated
+          run:
+            path: sh
+            args:
+              - -c
+              - |
+                cd git-repo-fortigate-autoscale-azure-updated
+                git fetch ../git-repo-fortigate-autoscale-azure main
+                git checkout ((gitrepo.head_branch)) || git checkout ((gitrepo.base_branch)) -b ((gitrepo.head_branch))
+                git reset --hard FETCH_HEAD
+      - task: npm-install-dependencies
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: node
+              tag: 14-buster
```

Comment:
could use 14-alpine since you are using separate tasks and don't need git anymore? (alpine is a much smaller image)

Reply:
can't do that here, the Azure sdk requires some other special environment libraries like python and libsecret. using 14-buster saves some time